external/storm-kafka-migration/src/main/java/org/apache/storm/kafka/migration/KafkaSpoutMigration.java

```diff
@@ -54,25 +55,30 @@
         private int zkRetryIntervalMs;
     }
 
+    /**
+     * Migrates offsets from the Zookeeper store used by the storm-kafka non-Trident spouts, to Kafka's offset store used by the
+     * storm-kafka-client non-Trident spout.
+     */
     public static void main(String[] args) throws Exception {
         if (args.length != 1) {
             System.err.println("Args: confFile");
+            return;
```

Comment:
You may want to exit with explicit return code: it will help scripts which leverages this module. Same for trident migration.